### PR TITLE
Enable FC tests

### PR DESCRIPTION
**Pull request by tugsbayasgalan**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #72479
* #72474

Differential Revision: [D34057581](https://our.internmc.facebook.com/intern/diff/D34057581)

### CODE REVIEW

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34057581).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34057581).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34057581).

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34057581).

File: test/forward_backward_compatibility/check_forward_backward_compatibility.py [hunk 1]
```diff
@@ -31,13 +31,18 @@
...
+    ("aten::_svd_helper", datetime.date(2022, 3, 1)),
+]
+
+# Same things as TEMPORARY__BC_ALLOW_LIST but for FC changes
+TEMPORARY_FC_ALLOW_LIST = [
     ("aten::_svd_helper", datetime.date(2022, 3, 1)),
 ]
 
 # WARNING: Operators included in this list indefinitely bypass all BC and FC schema checks.
 # This is almost certainly NOT what you want to do. See note above.
```
Comment (mruberry) [inline test/forward_backward_compatibility/check_forward_backward_compatibility.py:44]: Which note does this refer to? The note should be given a name

File: test/forward_backward_compatibility/check_forward_backward_compatibility.py [hunk 2]
```diff
@@ -31,13 +31,18 @@
 # ]
 #
 # NB: function name DOES NOT include overload name!
```
[the remark this answers is not in the document]
Comment (mruberry) [reply test/forward_backward_compatibility/check_forward_backward_compatibility.py:33]: We should also point to our BC/FC policy

File: test/forward_backward_compatibility/check_forward_backward_compatibility.py [hunk 3]
```diff
@@ -76,9 +81,50 @@
     "aten::_native_multi_head_self_attention",
 ]
 
-def compile_temp_allow_list():
+# Same thing as INDEFINITE_BC_ALLOW_LIST
+# but for FC changes.
+INDEFINITE_FC_ALLOW_LIST = [
```
Comment (mruberry) [inline test/forward_backward_compatibility/check_forward_backward_compatibility.py:86]: What's dangerous about adding an operator to this list?

**Comment by mruberry** [comment]
This stack looks OK but the lint and test failure appear real -- what's up with that?

**Review by gmagogsfm — APPROVED** [review]

**Comment by tugsbayasgalan** [comment]
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34057581).

**Pull request closed without merging**